DEBUGING.md

```diff
@@ -6,41 +6,44 @@ with the development of PEcAn.
 # ----------------------------------------------------------------------
 # debug options
 # ----------------------------------------------------------------------
-options(warn = 1, keep.source = TRUE, error =
-          quote({
-            status.end("ERROR")
+options(
+  warn = 1,
+  keep.source = TRUE,
+  error = quote(
+    tryCatch(
+      expr = {
+        status.end("ERROR")
```

Comment:
would it make sense to move the tryCatch into the status.end(). I would like the rest of the commands to run no matter what, like printing open connectins and the stack trace for debugging purposes.

Same below so we make sure the killTunnel is removed.

Reply:
How about a separate tryCatch for each function? Then we know we've *attempted* every step even if any one of them fails.

Reply:
@robkooper changed in a1a4ddab